plugins/chat/app/serializers/chat/direct_message_serializer.rb

```diff
@@ -5,7 +5,7 @@ class DirectMessageSerializer < ApplicationSerializer
     attributes :group, :users
 
     def users
-      users = object.direct_message_users.map(&:user).map { |u| u || Chat::NullUser.new }
+      users = object.users.map { |u| u || Chat::NullUser.new }
```

Comment:
https://github.com/discourse/discourse/blob/f51960287f6a7297243fd904966811c90b7ce5b1/plugins/chat/app/models/chat/direct_message.rb#L10-L13

`object.direct_message_users.map(&:user)` does the same thing as `object.users` here.